concordium-consensus/src/Concordium/External/GRPC2.hs

```diff
@@ -980,6 +983,26 @@ getLastFinalizedBlockSlotTimeV2 cptr = do
     Ext.ConsensusRunner mvr <- deRefStablePtr cptr
     runMVR Q.getLastFinalizedSlotTime mvr
 
+-- |Get the earliest time in which a baker wins the lottery.
+-- Returns "unavailable" for consensus version 0.
+-- For consensus version 1, it will return a result (even if the baker ID does not correspond
+-- to a valid baker).
```

Comment:
```suggestion
-- For consensus version 1, it will return a result (if the baker ID does not correspond
-- to a valid baker, then the result will be the start of the following reward period).
```
?